docs/panel-stable/js/pages/games/games.js

```diff
@@ -62,8 +62,8 @@ $(function () {
     $('#adventureSystemSettings').on('click', function () {
         socket.getDBValues('get_adventure_settings', {
             tables: ['adventureSettings', 'adventureSettings', 'adventureSettings', 'adventureSettings', 'adventureSettings', 'adventureSettings',
-                'adventureSettings', 'adventureSettings'],
-            keys: ['joinTime', 'coolDown', 'gainPercent', 'minBet', 'maxBet', 'enterMessage', 'warningMessage', 'coolDownAnnounce']
+                'adventureSettings', 'adventureSettings', 'adventureSettings'],
+            keys: ['joinTime', 'coolDown', 'gainPercent', 'minBet', 'maxBet', 'enterMessage', 'warningMessage', 'coolDownAnnounce', 'startPermission']
```

Comment:
Never edit the stable remote panel, this can break bots when this isn't even released. It is done via automation

Reply:
Reverted the change